doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,9 @@
 
 <h3>New features since last release</h3>
 
+* Added method `qml.math.sqrt_matrix_sparse` to compute the square root of a sparse matrix.
```

Comment:
Regarding the naming, is there a convention we are following here (numpy/scipy, other PL functionality...), and if not, should we be?

Reply:
I'm now aware of that yet. Up to now I've been naming all the sparse things by suffixing them `_sparse` since it's something drastically different from what we used to a.k.a. the TensorLike system.

Reply:
Sounds good, thanks :+1: